contrib/dockerd-rootless.sh

```diff
@@ -32,6 +32,10 @@ if ! [ -d "$HOME" ]; then
 	echo "HOME needs to be set and exist."
 	exit 1
 fi
+if [ "$(dockerd-rootless-setuptool.sh check | sed 's/.*\(Requirements are satisfied\).*/\1/')" != "Requirements are satisfied" ]; then
```

Comment:
`dockerd-rootless-setuptool.sh check` will set a non-zero exit code if the requirements are not satisfied.
I think it would be simpler and cleaner to check for the return code instead of checking the message.

Reply:
Also, this assumes that the `dockerd-rootless-setuptool.sh` is available in `$PATH`, which may not be the case.
I don't think we can even assume user has this script at all - so this check should be completely optional.

Reply:
I think the check shouldn't even be executed by default even if the script is available. IMO if we want the check, it should be opt-in with a flag. But then - why have a separate flag if the user can just do the check himself before calling the `dockerd-rootless.sh` like this:

`dockerd-rootless-setuptool.sh check && docker-rootless.sh ...`

Reply:
ok, i came up with this then: 
if [ -f $(which dockerd-rootless-setuptool.sh) ]; then
    output="$(dockerd-rootless-setuptool.sh check)"
    if [ $? -eq 1 ]; then
        echo "Some dependencies are not met."
    fi
    unset output
fi
I think it is good?